src/app/dashboard/competitions/[competitionId]/register-form/create/components/CreateCompetitionRegisterForm.tsx

```diff
@@ -156,6 +155,7 @@ export default function CreateCompetitionRegisterForm({ competition }: CreateCom
           />
 
           <p className="text-lg text-gray-500">Datos del participante</p>
+          <p className="text-sm text-gray-500"> * Climbup pide el RUT del participante para poder registrarlos en la competencia.</p>
```

Comment:
Recuerda la mayúscula en ClimbUp
```suggestion
          <p className="text-sm text-gray-500"> * ClimbUp pide el RUT del participante para poder registrarlos en la competencia.</p>
```